packages/react/README.md

```diff
@@ -15,68 +15,45 @@ Use `npm` to install the Parallel JS SDK module and this React library:
 
 ```js
 import { loadParallel } from '@parallelmarkets/vanilla'
 import { ParallelProvider } from '@parallelmarkets/react'
 
 // Start loading the parallel library with the given configuration information. Make sure
-// you call this outside of a component's render to avoid recreating a parallel object
-// on every render.
+// you call this outside of a component's render to avoid recreating a `Parallel` object
+// on every render. Also - you should not "await" the resulting promise, just pass directly
+// to the ParallelProvider in the "parallel" property
 const parallel = loadParallel({ client_id: '123', environment: 'demo', flow_type: 'overlay' })
 
 const App = () => (
   <ParallelProvider parallel={parallel}>
-    <AccreditationArea />
+    <YourRootComponent />
   </ParallelProvider>
 )
 
-const root = createRoot(document.getElementById('main'))
+const app = document.getElementById('main')
+const root = createRoot(app)
 root.render(<App />)
 ```
 
-Then, anywhere inside your application you can call the hook `useParallel` to get access to a number of useful properties:
+> [!NOTE]  
+> To best leverage Parallel's fraud detection, include the call to `loadParallel` across your app/site. This allows Parallel to detect suspicious behavior that may be indicative of fraud as users interact with your website.
 
- * `parallel`: This provides access to the [full JS SDK](https://developer.parallelmarkets.com/docs/javascript/sdk)
- * `loginStatus`: The current login status of the user. This is the result of a call to [`getLoginStatus()`](https://developer.parallelmarkets.com/docs/javascript/sdk), which may be null initially until a call to the API finishes.
- * `getProfile()`: A function that returns a `Promise` that will be resolved with profile information (from a call to the [Profile API](https://developer.parallelmarkets.com/docs/server/profile-api))
- * `getAccreditations()`: A function that returns a `Promise` that will be resolved with accreditation information (from a call to the [Accreditations API](https://developer.parallelmarkets.com/docs/server/accreditations-api))
- * `getBlockchain()`: A function that returns a `Promise` that will be resolved with blockchain information (from a call to the [Blockchain API](https://developer.parallelmarkets.com/docs/server/blockchain-api)) 
- * `getIdentity()`: A function that returns a `Promise` that will be resolved with identity KYC/AML information (from a call to the [Identity API](https://developer.parallelmarkets.com/docs/server/identity-api))
- * `login()`: A shortcut to `parallel.login()`
- * `logout()`: A shortcut to `parallel.logout()`
- * `error`: Any error that resulted from getting the current login status on load
```

Comment:
I think we could keep these as a way of listing all the properties available (and just remove `getAccreditation()` / `getIdentity()` / `getBlockchain()`).  WDYT?

Reply:
Yeah, good call. I integrated it back in to roughly where it was, after the introduction of `ParallelProvider`:

<img width="1037" alt="image" src="https://github.com/parallel-markets/parallel-js/assets/179257/dd0762d3-b213-4513-ae5f-99759e42cf3d">